spec/lib/vmdb/loggers_spec.rb

```diff
@@ -8,9 +8,20 @@
     log_file_path.delete if log_file_path.exist?
   end
 
+  def in_appliance_env(example)
+    old_env = ENV.delete('APPLIANCE')
+    ENV['APPLIANCE'] = 'true'
+    MiqEnvironment::Command.instance_variable_set(:@is_appliance, nil) # Clear the MiqEnvironment cache from application boot
```

Comment:
NOTE now that we check `MiqEnvironment::Command.is_appliance?/is_podified?` early on just changing the ENV vars isn't sufficient

Reply:
wonder if this should move into EvmSpecHelper.clear_caches

Reply:
Oh yeah, didn't know that existing but absolutely